core/status/viable.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package status
+
+// IsMachineViable returns true if the machine is started.
+func IsMachineViable(status StatusInfo) bool {
```

Comment:
What about just `IsMachinePresent` or `IsMachineConnected` and similar

Reply:
Present! Yes, much better. I'll use that.